nix/tools/bazel-watcher/default.nix

```diff
@@ -51,10 +51,7 @@ buildBazelPackage rec {
     '';
 
     sha256 =
-      if stdenv.isDarwin then
-        "0nz0z9d8y7kckgzshlsbjxrkqdaxrjdyinjs4ffkbvr3iikf958g"
-      else
-        "08yr8419yl065sf129xy1dzadsxgq0vwsl59x7wq6f0ax08si1di"
+        "151s3k2phkdb32yxdja1q0kbxgb4alz8l9wfyzgd2mbdrkq4a62q"
```

Comment:
I don’t really know why this is no longer platform dependent but I also never understood why it was platform dependent in the first place :woman_shrugging: